app/views/directives/deployment-metrics.html

```diff
@@ -45,7 +45,7 @@
     </p>
   </div>
 
-  <div ng-repeat="metric in metrics" ng-if="!noData && !metricsError" class="metrics-full">
+  <div ng-repeat="metric in metrics" ng-show="!noData && !metricsError" class="metrics-full">
```

Comment:
The div with the chart ID has to exist before calling c3.generate. It worked with ng-if before because we called $timeout in the controller, but this PR removes the $timeout.